### ISSUE TITLE
Getting error as "failure: null"

### ISSUE THREAD

**Issue by mastery-sanket-nagare** [adevinta/zoe#29]
When I try to run any Zoe command getting the following error 
failure: null

ex:
$ zoe -c dev -e dev topics list
2021-04-07 09:30:19 INFO zoe: loading config from url : file:/home/sanket_nagare/.zoe/config/dev.yml
2021-04-07 09:30:20 INFO zoe: requesting topics...
failure: null

**Comment by wlezzar** [comment]
Hi @mastery-sanket-nagare . Thanks for raising the issue!

Can you please run the same command with `ZOE_STACKTRACE=1`? Something like this: 
```
ZOE_STACKTRACE=1 zoe -c dev -e dev topics list
```

Also it would be help me if you provide us with your config file:

```
cat /home/sanket_nagare/.zoe/config/dev.yml
```

**Comment by mastery-amit-sahani** [comment]
@wlezzar I'm getting the same error when trying to consume from a topic, where as consuming from another topic within same environment works fine.

` ZOE_STACKTRACE=1 zoe -e dev topics consume test -n 1 --from P1D --with-meta --continuously
2021-06-16 19:59:17 INFO zoe: loading config from url : file:/home/amit/.zoe/config/dev.yml
2021-06-16 19:59:18 INFO zoe: querying offsets...
2021-06-16 19:59:31 INFO zoe: polling topic 'test' (subscription : AssignPartitions(partitions={0=268, 3=348, 4=557, 1=787, 2=510, 7=284, 5=309, 6=412}))
failure: null
java.lang.StackOverflowError
        at java.base/java.lang.StringBuilder.<init>(Unknown Source)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:36)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)
        at com.adevinta.oss.zoe.service.runners.BaseKt.fromRunFailureResponse(base.kt:40)`

**Comment by wlezzar** [comment]
Hi @mastery-amit-sahani ! 

Thanks a lot, this is super useful. It's clearer where the issue comes from. I will produce a fix for this issue and keep you updated.

**Comment by wlezzar** [comment]
@mastery-amit-sahani I pushed the fix in the version `0.27.2`. Can you try that version out and give some feedback?

If you have installed zoe with `brew`, you can just use: `brew upgrade adevinta/homebrew-zoe/zoe`

**Comment by mastery-amit-sahani** [comment]
Thanks @wlezzar with the new version at-least now the error is relevant.
